spec/converterbase_spec.rb

```diff
@@ -293,6 +293,105 @@ def to_ruby(match, m1, m2, open_close)
                "追撃をしない俺に振り返り、｜再度《パワースラッシュ 》"
       end
     end
+
+    describe "｜を使ったルビ位置の指定" do
+      it "《 》基本" do
+        expect(to_ruby( \
+               "｜山田太郎《やまだたろう》",
+               "｜山田太郎", "やまだたろう", ["《", "》"])).to eq \
+               "｜山田太郎《やまだたろう》"
+      end
+      it "（ ）基本" do
+        expect(to_ruby( \
+               "｜山田太郎（やまだたろう）",
+               "｜山田太郎", "やまだたろう", ["（", "）"])).to eq \
+               "｜山田太郎《やまだたろう》"
+      end
+      it "《 》ルビを振りたくない場合" do
+        expect(to_ruby( \
+               "山田太郎｜《やまだたろう》",
+               "山田太郎｜", "やまだたろう", ["《", "》"])).to eq \
+               "山田太郎《やまだたろう》"
+      end
+      it "（ ）ルビを振りたくない場合" do
+        expect(to_ruby( \
+               "山田太郎｜（やまだたろう）",
+               "山田太郎｜", "やまだたろう", ["（", "）"])).to eq \
+               "山田太郎（やまだたろう）"
+      end
+    end
+
+    describe "ルビの自動対応" do
+      it "（ ）自動" do
+        expect(to_ruby( \
+               "HP（ヒットポイント）",
+               "HP", "ヒットポイント", ["（", "）"])).to eq \
+               "｜HP《ヒットポイント》"
+      end
+      it "《 》自動" do
+        expect(to_ruby( \
+               "ＭＰ《マジックポイント》",
+               "ＭＰ", "マジックポイント", ["《", "》"])).to eq \
+               "｜ＭＰ《マジックポイント》"
+      end
+      it "（ ）漢字と認識されない1" do
+        expect(to_ruby( \
+               "久々（ひさびさ）",
+               "久々", "久々（ひさびさ）", ["（", "）"])).to eq \
+               "久々（ひさびさ）"
+      end
+      it "（ ）漢字と認識されない2" do
+        expect(to_ruby( \
+               "二〇八（ふたまるはち）",
+               "二〇八", "ふたまるはち", ["（", "）"])).to eq \
+               "二〇｜八《ふたまるはち》"
+      end
+      it "（ ）ルビ化されない" do
+        expect(to_ruby( \
+               "山田太郎（山田太郎）",
+               "山田太郎", "山田太郎", ["（", "）"])).to eq \
+               "山田太郎（山田太郎）"
+      end
+      it "《 》ルビ化されない" do
+        expect(to_ruby( \
+               "山田太郎《山田太郎》",
+               "山田太郎", "山田太郎", ["《", "》"])).to eq \
+               "山田太郎《山田太郎》"
+      end
+    end
+
+    describe "スペースを含む場合の特殊処理" do
+      it "（ ）の中にスペースを一つ含む場合" do
+        expect(to_ruby( \
+               "山田　太郎（やまだ　たろう）",
+               "山田　太郎", "やまだ　たろう", ["（", "）"])).to eq \
+               "｜山田《やまだ》　｜太郎《たろう》"
+      end
+      it "（ ）ルビにスペースがない場合" do
+        expect(to_ruby( \
+               "山田　太郎（やまだたろう）",
+               "山田 太郎", "やまだたろう", ["（", "）"])).to eq \
+               "山田 ｜太郎《やまだたろう》"
+      end
+      it "（ ）文字列にスペースがない場合" do
+        expect(to_ruby( \
+               "山田太郎（やまだ　たろう）",
```

Comment:
__[rubocop]__
* Layout/FirstParameterIndentation: Indent the first parameter one step more than `to_ruby( \`.

<!-- Comment by [SideCI](https://sideci.com) -->